lambdas/thumbnail/tests/conftest.py

```diff
@@ -16,6 +16,12 @@ def pytest_addoption(parser):
 
 
 def pytest_configure(config):
+    config.addinivalue_line(
+        "markers", "poppler: mark a test to run only if --poppler is indicated (poppler tools is installed)"
+    )
+    config.addinivalue_line(
+        "markers", "loffice: mark a test to run only if --loffice is indicated (LibreOffice is installed)"
+    )
```

Comment:
Nice, please submit as separate PR.

Reply:
Separated out into another branch.  I'll make a PR for it as soon as this goes through.